### ISSUE TITLE
Implement New Type Scales

### ISSUE THREAD

**Issue by johngruen** [mozilla/protocol#344]
Blocked on design...we need basic typescales for title and paragraph text

**Comment by stephaniehobson** [comment]
Basic typescales: https://github.com/mozilla/protocol-sketch-plugin/blob/master/src/data/firefox/text.json

**Comment by stephaniehobson** [comment]
Closing in favour of #342
